src/components/GPSButton/index.js

```diff
@@ -9,7 +9,7 @@ import LoadingIndicator from 'components/LoadingIndicator';
 import GPS from '../../shared/images/icon-gps.svg';
 
 const StyledButton = styled(Button)`
-  outline: 2px solid rgb(0, 0, 0, 0.1);
+  box-shadow: 0 0 0 2px rgb(0 0 0 / 10%);
```

Comment:
Can we use here `rgba(0, 0, 0, 0.1);` for consistency?

Reply:
I mean `rgba` instead `rgb` or else the last parameter is not used

Reply:
Changed it to `rgba` instead of `rgb` to keep consistency. 

`rgba` is an alias for `rgb` so the result stays the same :)

Reply:
:-) didn't know dat!!